pym/portage/data.py

```diff
@@ -142,6 +151,32 @@ def _get_global(k):
 		elif _get_global('portage_gid') in os.getgroups():
 			v = 1
 
+	elif k in ('_build_gid', '_build_uid'):
+		keyerror = False
+		try:
+			build_uid = pwd.getpwnam(_get_global('_build_user')).pw_uid
+		except KeyError:
+			writemsg('!!! PORTAGE_BUILD_USER refers to non-existant user "%s"\n'
```

Comment:
s/non-exist**a**nt/nonexist**e**nt/
(And 7 lines below)

Reply:
Thanks, fixed.